lib/vsc/utils/generaloption.py

```diff
@@ -1116,7 +1116,7 @@ def parseconfigfiles(self):
                     self.log.debug("parseconfigfiles: configfile %s not found, will be skipped" % fn)
 
             if fn in option_ignoreconfigfiles:
-                self.log.debug("parseconfigfiles: configfile %s will be ignored %s" % fn)
+                self.log.debug("parseconfigfiles: configfile %s will be ignored" % fn)
```

Comment:
or actually
`self.log.debug("parseconfigfiles: configfile %s will be ignored",fn)`